bpython/test/test_autocomplete.py

```diff
@@ -245,12 +245,12 @@ def test_att_matches_found_on_old_style_instance(self):
                                              locals_={'a': OldStyleFoo()}),
                             set(['a.method', 'a.a', 'a.b']))
         self.assertIn(u'a.__dict__',
-                      self.com.matches(3, 'a._', locals_={'a': OldStyleFoo()}))
+                      self.com.matches(3, 'a.__', locals_={'a': OldStyleFoo()}))
```

Comment:
If I remember correctly, the first argument should be changed to 4. @thomasballinger, can you confirm that?